cmd/nerdctl/run_test.go

```diff
@@ -245,3 +246,45 @@ func TestRunWithLogOpt(t *testing.T) {
 		}
 	}
 }
+
+func TestRunWithJournaldLogDriver(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("journald log driver is not yet implemented on Windows")
+	}
+	base := testutil.NewBase(t)
+	containerName := testutil.Identifier(t)
+
+	defer base.Cmd("rm", "-f", containerName).AssertOK()
+	base.Cmd("run", "-d", "--log-driver", "journald", "--name", containerName, testutil.CommonImage,
+		"sh", "-euxc", "hexdump -C /dev/urandom | head -n1000").AssertOK()
+
+	time.Sleep(3 * time.Second)
+	inspectedContainer := base.InspectContainer(containerName)
+
+	syslogIdentifier := fmt.Sprintf("%s:%s", testutil.Namespace, inspectedContainer.ID)
+	jconf := sdjournal.JournalReaderConfig{
+		Matches: []sdjournal.Match{
+			{
+				Field: sdjournal.SD_JOURNAL_FIELD_SYSLOG_IDENTIFIER,
+				Value: syslogIdentifier,
+			},
+		},
+	}
+
+	jr, err := sdjournal.NewJournalReader(jconf)
```

Comment:
Can we just exec `journalctl` to remove cgo dependency?

Reply:
I think journalctl is not enough if we want to read logs of what we have written.

I think it would be better to we can design the journal logger as an unreadable logger, and use `JSON` logger  and journal logger at the same time

Reply:
Why journalctl is not enough?
If journalctl is not enough, how is this expected to be used?

Reply:
Journalctl is possible with  SYSLOG_IDENTIFIER   filter. need for cgo when using journactl in tests. 
@Zheaoli I want to avoid making depenciences between drivers